content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst

```diff
@@ -87,18 +87,24 @@ The inventory valuation report
 ------------------------------
 
 To view the current value of all products in the warehouse, go to :menuselection:`Inventory app -->
-Reporting --> Inventory Valuation`. The records in the table are organized by product, and selecting
-a product's drop-down menu displays detailed records with the :guilabel:`Date`,
-:guilabel:`Quantity`, and :guilabel:`Total Value` of the inventory.
+Reporting --> Valuation`. The page displays detailed records of products with the :guilabel:`Date`,
+:guilabel:`Quantity`, :guilabel:`Unit Value` and :guilabel:`Total Value` of the inventory.
+
+.. important::
+   :ref:`Developer mode<developer-mode>` must be turned on to see the :guilabel:`Valuation` option
+   under :guilabel:`Reporting`.
```

Comment:
```suggestion
.. important::
   :ref:`Developer mode<developer-mode>` **must** be turned on to see the :guilabel:`Valuation`
   option under :guilabel:`Reporting`.
```
added ** to emphasize the importance of this step

this is a great note - but perhaps it could be more useful if its incorporated into the navigation steps, as it took me a second to realize I needed to be in Developer Mode before my eyes found this notation - just a suggestion. totally your call.